index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <div class="container" id="main">
+      <h1 id="title"> Dr. Norman Borlaug </h1>
+      <div class="main-img" id="img-div">
+        <img src="images/main.jpg" alt="main-image" id='image'>
+        <p id = 'img-caption'>Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger. </p>
+      </div>
+      <div id="tribute-info">
```

Comment:
Same goes here, using a <section> tag for <div> would have been better.

Reply:
Kindly explain why you suggested this comment, it might be obvious to you but not to them 😄 👍

Reply:
Great work so far. ✔
 But as I explained earlier, the importance of using semantic tags. Similarly using a semantic tag (section) in this scenario would be better. If you have still have any issues or difficulty on where to use semantic tags, please feel free to leave a comment here. 😎